examples/room/lod/lodmanager/manager.go

```diff
@@ -0,0 +1,289 @@
+// © 2019-2025 Diarkis Inc. All rights reserved.
+
+package lodmanager
+
+import (
+	"fmt"
+	"maps"
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"github.com/Diarkis/diarkis/packet"
+	"github.com/Diarkis/diarkis/user"
+)
+
+// sendMessage represents a message to be sent to a user
+type sendMessage struct {
+	receiverUserID string
+	ver            uint8
+	cmd            uint16
+	payload        []byte
+}
+
+// Manager manages LOD (Level of Detail) synchronization for users
+type Manager struct {
+	userEntities          map[string]*UserEntity
+	started               atomic.Bool
+	ver                   uint8
+	cmd                   uint16
+	syncIntervalForNearby int32
+	syncIntervalForFar    int32
```

Comment:
I think this should already be pre-calculated within the struct on initialization, rather than recalculating it everywhere else.
```suggestion
	syncIntervalForNearbyMs time.Duration
	syncIntervalForFarMs    time.Duration
```

Reply:
That might indeed be better. However, since time.Duration itself is not dependent on whether it's milliseconds or not, I will keep the existing name.